adobe/spectrum-web-components
# chore: demo controls should reflect for all attributes

…flected



## Description

In the Demo section of the Documentation, changes are reflected only for attributes which are controlled by a `sp-picker`, rest are being ignored.
I also added some CSS so that all controls have the same width (this is optional, I can revert if it is not needed).

## Related issue(s)



- N/A

## Motivation and context


Small documentation issue.

## How has this been tested?




    1. Go [here](https://opensource.adobe.com/spectrum-web-components/components/alert-banner/)
    2. Change the text in the `text` input
    3. Observe nothing happens

    1. Go [here](https://opensource.adobe.com/spectrum-web-components/components/contextual-help/)
    2. Change the text in the `offset` input
    3. Observe that the property is not applied
    
    1. Go [here](https://rocss-docs-control-fix--spectrum-web-components.netlify.app/components/alert-banner/)
    2. Change the text in the `text` input
    3. Observe the component's text is updated

    1. Go [here](https://rocss-docs-control-fix--spectrum-web-components.netlify.app/components/contextual-help/)
    2. Change the number in the `offset` input
    3. Observe the component's ofset is updated


## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
